lesson_02/schemas/user.schema.js

```diff
@@ -0,0 +1,25 @@
+import Joi from '@hapi/joi';
+
+export const addUserSchema = Joi.object({
+    id: Joi.string().required(),
+    login: Joi.string().required(),
+    password: Joi.string().pattern(new RegExp('[a-zA-Z0-9]')).required(),
+    age: Joi.number().integer().min(4).max(130).required(),
+    isDeleted: Joi.boolean().required()
+});
+
+export const updateUserSchema = Joi.object({
+    id: Joi.any(),
+    login: Joi.string().required(),
```

Comment:
let's allow only numbers and letters for login

Reply:
Done